### DISCUSSION TITLE
AttributeError: 'list' object has no attribute '__module__' when calling from_pretrained using file system

### DISCUSSION THREAD

**Discussion by dferguson992** [huggingface/diffusers#7610] [Q&A]
I downloaded the stabilityai/stable-cascade model from HuggingFace and saved it to my file system using the following:
```python
model = StableCascadeCombinedPipeline.from_pretrained(HF_MODEL_ID, 
                                                      variant="bf16",
                                                      torch_dtype=torch.float16)
model_path = "model/"
code_path = "code/"

if not os.path.exists(model_path):
    os.mkdir(model_path)
if not os.path.exists(code_path):
    os.mkdir(code_path)

model.save_pretrained(save_directory=model_path)
```

My plan is to deploy this to a SageMaker endpoint but am having trouble re-loading the model from the local file system.  I keep getting the following error when I call `model = StableCascadeCombinedPipeline.from_pretrained("model/", variant="bf16", torch_dtype=torch.float16)`:
<img width="1163" alt="image" src="https://github.com/huggingface/diffusers/assets/5060436/7926fc03-e720-4d69-8796-8f1dad73f154">

Been at this for several days now, not sure what I'm doing wrong.

**Comment by tolgacangoz** [comment]
As of my investigation, one can add this (as a fast and temporary action) at the 506. line of `src/diffusers/pipelines/pipeline_loading_utils.py`:
```diff
     not_compiled_module = _unwrap_model(module)
+    if isinstance(not_compiled_module, list): return (not_compiled_module[0], not_compiled_module[1])
     library = not_compiled_module.__module__.split(".")[0]
